localstack-core/localstack/services/lambda_/event_source_mapping/pollers/dynamodb_poller.py

```diff
@@ -105,5 +105,8 @@ def get_approximate_arrival_time(self, record: dict) -> float:
         # TODO: parse float properly if present from ApproximateCreationDateTime -> now works, compare via debug!
         return record["dynamodb"].get("todo", datetime.utcnow().timestamp())
```

Comment:
any idea whether that one is related to the ms vs no-ms granularity?

Reply:
I saw that DynamoDB often returns timestamps in [unix epoch format](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_TableDescription.html#DDB-Type-TableDescription-CreationDateTime). Perhaps it has to do with that?